Models/XGBoosting.py

```diff
@@ -66,7 +66,7 @@ def submit_solution(self):
         submission = test[['Id']]
         submission['SalePrice'] = predictions['SalePrice']
         submission.to_csv(
-            "../Submissions/submission_XGB_best_estimator7_" + str(
+            "../Submissions/submission_XGB_best_estimator9_" + str(
```

Comment:
Actually I did not add it to the staging env. I thought since it was not the best submission so it may not be important to add it.  
Will update the PR. I will add the 2 results which I submitted on Kaggle. There were a few other files which I didn't submit to Kaggle as they had high rmse. So will not add them to the staging env.